quantum/nodes/ibm-quantum-system/ibm-quantum-system.html

```diff
@@ -0,0 +1,71 @@
+<script type="text/javascript">
+  RED.nodes.registerType("ibm-quantum-system", {
+    category: "quantum",
+    color: "#A8A8A8",
+    defaults: {
+      name: { value: "" },
+      api_token: { value: "", required: true },
+      preferred_backend: { value: "", required: true },
```

Comment:
Currently, the backend is selected by entering a string and is always required. Is it possible to do two things with this:
- Instead of entering a string, provide a drop-down list which contains a list of possible backends. This ensures that the user doesn't incorrectly enter the name of a backend, and its made clear exactly what is available to them.
- Instead of having the field as required, set a default backend that the user can use if they don't select one themselves. For example, perhaps we could use the [backends function](https://qiskit.org/documentation/stubs/qiskit.providers.ibmq.AccountProvider.html#qiskit.providers.ibmq.AccountProvider.backends) for retrieving a list of all backends available to the user, and then use the [least_busy function](https://qiskit.org/documentation/stubs/qiskit.providers.ibmq.least_busy.html#qiskit.providers.ibmq.least_busy) for selecting a default backend.

Reply:
I intend to find a better way of doing this but thought it best to merge the working version first so that there is a node available to work with while I work on implementing the other features. I mentioned that in the description of the PR.

Reply:
Ah my bad, I forgot you put that in the PR description